Kajabi/sage-archived
# Typography style refactor

## Description
Headlines and overall typography are currently styled by their element names (`h1, h2, h3`, etc). By using classes for styling these elements, we'll be better able to maintain semantic HTML separate from styling, and prevent overrides of unintended margins, padding, etc.

- [x] Restructure heading styles to use classes
- [x] Create `letter-spacing` token
- [x] Globalize base type styles for theming
- [x] Update styles throughout documentation to use new classnames
- [x] Modify documentation in Sage around type and its use
- [x] Add documentation in Guidelines regarding adding type specs

## Test notes

### Steps for testing
No visual changes should be introduced with this refactor.
1. Start Sage locally
2. Navigate to the [Typography](http://localhost:4000/sage/pages/typography) overview page
3. Compare styles with Figma to verify spec remains the same

[Documentation in Guidelines](https://github.com/Kajabi/sage/wiki/Understanding-and-Maintaining-Type-Specs)
